src/main/java/at/hannibal2/skyhanni/data/PartyAPI.kt

```diff
@@ -15,6 +15,10 @@ object PartyAPI {
     // TODO USE SH-REPO
```

Comment:
We shuld use repopatterns for the whole class. not sure if this pr is the correct spot though

Reply:
Please not <3

Reply:
then we are on the same page here, nice.